seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/TransactionStateFileSinkWriter.java

```diff
@@ -50,7 +50,7 @@ public class TransactionStateFileSinkWriter implements SinkWriter<SeaTunnelRow,
 
```

Comment:
Please keep this file change out of your submission, give you a advice, you can revert this file and keep this file the same as dev branch.